### ISSUE TITLE
Is there an option to not log skipped tests to console when using dotnet test

### ISSUE THREAD

**Issue by nguerrera** [microsoft/vstest#10453]
I would like to suppress all of the following output (most of it yellow) when running `dotnet test`:
```
[xUnit.net 00:00:31.15]     TestName [SKIP]
  Skipped TestName [1 ms]
```

Can this be done? I've searched high and low before creating this issue with no luck.

Also note that I don't want to turn down the verbosity so low that I miss other important things, I just don't want to hear all the skipped test details every time I run.

**Comment by nohwnd** [comment]
There is no option to not show just the skipped tests, there is also no plan to add it.

**Comment by nohwnd** [comment]
Yes. But can we first agree on the design and how to show the option to the user?

**Comment by nohwnd** [comment]
I can accept a PR but we need to design it first, and hopefully get some upvotes on this issue, or a good motivation to add it. Maybe it would be possible for you to migrate to our new testing.platform? (I don't think we have that option there either, but there is a better chance of adding it.)

**Comment by nohwnd** [comment]
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?tabs=dotnetcli

**Comment by nguerrera** [comment]
I will look into the new platform. This is the first I've heard of it. :) Then I'll come back and talk about motivation and design. :)
